spec/label_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './spec/spec_helper'
+require 'date'
+describe Label do
+  before :all do
+    @label = Label.new('Vampire Diaries', 'red')
+  end
+
+  describe '#new' do
+    it 'takes in three parameters and returns a book object' do
```

Comment:
[Optional] Please check the title of this test, Maybe you mean the `Label` class instead of `Book` class